draft-ietf-quic-multipath.md

```diff
@@ -481,6 +481,9 @@ it SHOULD NOT use the associated Path ID in future frames, except
 in ACK_MP frames for inflight packets and
 in MP_RETIRE_CONNECTION_ID frames for connection ID retirement.
 
+After a path is abandoned, the Path ID MUST NOT be reused
+for new paths, as the Path ID is part of the nonce calculation {{multipath-aead}}.
+
```

Comment:
Copying as suggestion what Christian suggests. Note that I am not sure that we need to provide the example of "more than half of the available Path ID are used".

```suggestion

Endpoint SHOULD send MAX_PATHS frames {{max-paths-frame}} to raise
the limit of Path ID when endpoint finds there are not enough unused
Path ID (e.g. more than half of the available Path ID are used).
```